### DISCUSSION TITLE
编译ax3600固件报错configure: error: in `/home/zcluo/openwrt-boos/build_dir/target-aarch64_cortex-a53_musl/elfutils-0.186': configure: error: failed to find fts_close

### DISCUSSION THREAD

**Discussion by zcluo** [Boos4721/openwrt#63] [Bugs/问题反馈]
configure: error: in `/home/zcluo/openwrt-boos/build_dir/target-aarch64_cortex-a53_musl/elfutils-0.186':
configure: error: failed to find fts_close
See `config.log' for more details
make[3]: *** [Makefile:118: /home/zcluo/openwrt-boos/build_dir/target-aarch64_cortex-a53_musl/elfutils-0.186/.configured_a17fb5ef857664f03cd0ce37cc5ea591] Error 1
make[3]: Leaving directory '/home/zcluo/openwrt-boos/package/libs/elfutils'
time: package/libs/elfutils/compile#5.42#0.76#7.03
    ERROR: package/libs/elfutils failed to build.
make[2]: *** [package/Makefile:116: package/libs/elfutils/compile] Error 1

**Comment by Boos4721** [comment]
**check your config**

**Comment by fichenx** [comment]
我也遇到了，我是AX6，就算是使用默认配置也会出现这个错误。

**Reply by zjwxh** [reply]
我昨天晚上编译AX6正常，以前也正常，检查你自己原因吧

**Comment by lanyirenjg** [comment]
同样AX3600，编译也遇到这个问题了，一模一样的报错。不知道应该检查哪个配置

**Comment by jliveusa** [comment]
1、libraries中选择musl-fts
2、单独编译musl-fts
3、继续make -j1

**Reply by flyfeel** [reply]
老哥 你这说的第2点 单独编译这个mssl-fts  是什么意思？ 具体用什么命令 怎么操作？纯小白不懂哈，只会照着教程里的命令复制粘贴

**Reply by chika0801** [reply]
ax3600 一样出错。感谢楼主能说详细点吗
